presto-main/src/main/java/com/facebook/presto/Session.java

```diff
@@ -99,6 +102,8 @@ public final class Session
     private final OptimizerInformationCollector optimizerInformationCollector = new OptimizerInformationCollector();
     private final OptimizerResultCollector optimizerResultCollector = new OptimizerResultCollector();
     private final CTEInformationCollector cteInformationCollector = new CTEInformationCollector();
+    private final Map<PlanNodeId, PlanNodeStatsEstimate> planNodeEstimateMap = new HashMap<>();
```

Comment:
call this planNodeStatsMap and rename the getter.  Makes it parallel to planNodeCostMap.